format-specs/schema.json

```diff
@@ -74,7 +74,9 @@
             },
             "covering": {
               "type": "object",
-              "minProperties": 1,
+              "required": [
+                "bbox"
+              ],
```

Comment:
I don't think we want to make this required if we want to add other covering types in the future. Or would we remove this `required` when we add the future covering type?

Reply:
Or is there some way to define the specific key names allowed? E.g. that today we only allow the "bbox" key inside `coverings`, but that in the future other keys may be allowed as well?

Reply:
Yeah, that's the question. I mean right now it's required, isn't it? Otherwise it's empty or someone puts anything else.

In the future the requirement could be relaxed of course, but for 1.1 it doesn't feel very useful to not require it?!

With regards to your second comment, I'm not sure I understand what you are looking for.

Reply:
> With regards to your second comment, I'm not sure I understand what you are looking for.

My understanding is that your change means that a `bbox` key is now required. Instead, is there some way in JSON schema to allow an enum of values? So we currently define an enum with a single variant but can expand that in the future?

Reply:
You could do something like the following to indicate that you expect more fields in the future where having one of them is required:

```
"anyOf": [
  { "required": ["bbox"] }
]
```

It's not really different though, it's just an implicit indication for people reading the schema.

The question really is:

Are `covering: {}` or e.g. `covering: { a: "b" }` something that should be allowed in 1.1? Or do we always expect that in 1.1 there is a bbox member in covering?

Reply:
I think the expectation is that if the `covering` key exists, then it has a `bbox` member key.

Reply:
Then the PR is correct. If you want I can update it to the anyOf, I'm pretty neutral on it. What do others think?

Reply:
Agreed. I think it's fine as-is. Whenever more options are added, the `"required": ["bbox"]` can change to anyOf or oneOf without losing the meaning. And since only the outermost element has additionalProperties set to false, it seems consistent to leave it out here.